src/Core/Form/ChoiceProvider/ModuleByNameChoiceProvider.php

```diff
@@ -55,7 +56,7 @@ public function getChoices()
     {
         $moduleChoices = [];
 
-        /** @var ModuleInterface $module */
+        /** @var Module $module */
```

Comment:
Here the original phpDoc could be kept as it was only if `ModuleInterface` was properly defined.